lib/Cake/Network/CakeResponse.php

```diff
@@ -1336,7 +1336,7 @@ public function file($path, $options = array()) {
 			'download' => null
 		);
 
-		if (strpos($path, '..') !== false) {
+		if (strpos($path, '../') !== false || strpos($path, '..\\') !== false) {
```

Comment:
The thing is `/` can work in path on windows too, so better to keep it this way.